samples/Directory.Build.targets

```diff
@@ -1,14 +1,5 @@
 <Project>
   <ItemGroup>
-    <PackageReference Update="Xamarin.Android.Glide.Annotations" Version="4.12.0.8" />
-    <PackageReference Update="Xamarin.Android.Glide.DiskLruCache" Version="4.12.0.8" />
```

Comment:
Directory.Build.targets in samples folder is regenerated based on `config.json`, so that the latest not-published packages are used for testing with samples. This file could have been ignored too. 
I think this noise is caused by manually resolving merge conflicts because of differences in PRs